dask/bag/tests/test_bag.py

```diff
@@ -190,6 +190,16 @@ def test_can_use_dict_to_make_concrete():
     assert isinstance(dict(b.frequencies()), dict)
 
 
+def test_from_url():
+    a = db.from_url(['http://google.com', 'http://github.com'])
+    assert a.npartitions == 2
+    a.compute()
```

Comment:
I've found that having lines as items in the bag rather than characters also tends to be more useful in applications.